recipes/neo4jsbml/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "neo4jsbml" %}
+{% set version = "0.10.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/brsynth/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 172d8ab08cc9b4c40eca3a3b9871727fb1feec635ea368630e3e3456bfd11708
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install -vv .
+  entry_points:
+    - "neo4jsbml=neo4jsbml.__main__:main"
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - python >=3.8
+    - pyyaml
+    - setuptools
+  run:
+    - python >=3.8
+    - python-libsbml
+    - neo4j-python-driver
+    - sphinx
+    - sphinx_rtd_theme
+
+test:
+  imports:
+    - {{ name }}
+  commands:
+    - {{ name }} --help
+
+about:
+  home: https://github.com/brsynth/{{ name }}
+  summary: Import SBML file into Neo4j
+  license: MIT
+  license_file: LICENSE
+  doc_url: https://{{ name }}.readthedocs.io/
+  dev_url: https://github.com/brsynth/{{ name }}
```

Comment:
Can you remove the templating here? It makes it very hard to copy/paste the URLs for testing.

Reply:
fix 6cf4f5b1c6